.github/workflows/java-continuous-integration.yml

```diff
@@ -10,6 +10,31 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
+        scala-version: [2.11, 2.12]
+        spark-version: [2.4.7, 2.4.8]
+        include:
+          - os: ubuntu-latest
+            scala-version: 2.12
+            spark-version: 3.0.3
+          - os: ubuntu-latest
+            scala-version: 2.12
+            spark-version: 3.1.1
+          - os: ubuntu-latest
+            scala-version: 2.12
+            spark-version: 3.1.3
+          # build failures, see https://app.clickup.com/t/2ae2ree
+          #- os: ubuntu-latest
+          #  scala-version: 2.12
+          #  spark-version: 3.2.0
+          #- os: ubuntu-latest
+          #  scala-version: 2.12
+          #  spark-version: 3.2.1
+          #- os: ubuntu-latest
+          #  scala-version: 2.13
+          #  spark-version: 3.2.0
+          #- os: ubuntu-latest
+          #  scala-version: 2.13
+          #  spark-version: 3.2.1
```

Comment:
Let's just remove these lines. You can keep some of the changes in a separate branch as we are expanding the versions we build.